Common/Logger.cs

```diff
@@ -23,16 +23,16 @@ public Logger(IFileWriteHandler FileWriteHandler, IEnvironmentVariables environm
             logFileName = $"{DateTime.Now.Day}.{DateTime.Now.Month}.{DateTime.Now.Year}.{DateTime.Now.Hour}.{DateTime.Now.Minute}.{DateTime.Now.Second}";
         }
 
-        location = $"bin/pirate{version}/logs";
         version = environmentVariables.GetVariable("version");
+        location = $"bin/pirate{version}/logs";
     }
 
     public bool Log(string message, string orginFile, LogType logType)
     {
         var time = DateTime.Now.ToString();
         var text = ($"{time.Replace(" uur", "")}: {logType.ToString()}: {orginFile}.cs: {message}\n");
 
-        _fileWriteHandler.AppendToFile(new FileWriteModel("log", ".log", location, text));
+        _fileWriteHandler.AppendToFile(new FileWriteModel(logFileName, FileExtension.LOG, location, text));
```

Comment:
Make fileExtension enum lower caps....then in the filehandlers it doesnt need to be lowercased